exercises/hangman/Example.cs

```diff
@@ -1,85 +1,91 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
 using System.Linq;
-
-public delegate void HangmanChangedEventHandler(object sender, HangmanState state);
-
+using System.Reactive;
+using System.Reactive.Subjects;
+
+/// <summary>
```

Comment:
I really like that you made this class immutable! This comment is very useful to me as a reviewer, but less useful to student I think. As `hangman` is one of the last exercises, a student should at this point be quite familiar with (im)mutability. Therefore, I propose to remove this comment.